.travis.yml

```diff
@@ -22,7 +22,6 @@ matrix:
 
 before_script:
   - phpenv config-add ci/travis.php.ini
-  - composer remove --dev spryker/git-hook
```

Comment:
this is only valid if you also remove the source of the problem.. https://github.com/spryker-sdk/spryk/blob/master/composer.json#L24

Reply:
if tests pass, then never mind.